github-issues.go

```diff
@@ -19,23 +19,18 @@ package main
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
 	"net/http"
-	"time"
-)
+	"strings"
 
-// Label - holds issue labels
-type Label struct {
-	Name  string `json:"name"`
-	Color string `json:"color"`
-}
+	"github.com/google/go-github/github"
+)
 
-// GitIssues - holds only relevant issues
-type GitIssues struct {
+// GitIssue - holds only relevant issues
+type GitIssue struct {
 	Number    int    `json:"number"`
 	Title     string `json:"title"`
 	Link      string `json:"html_url"`
-	Labels    []Label
+	Labels    []github.Label
```

Comment:
Yes, their library has JSON fields on it. We can reuse all of their structures actually — we should talk about this.